docs/pages/enroll-resources/agents/join-services-to-your-cluster/oracle.mdx

```diff
@@ -0,0 +1,112 @@
+---
+title: Join Services with Oracle Cloud
+description: Use the Oracle join method to add services to your Teleport cluster.
+---
+
+This guide will explain how to use the **Oracle join method** to configure
+Teleport processes to join your Teleport cluster without sharing any secrets
+when they are running in an Oracle Cloud Infrastructure (OCI) Compute instance.
+
+The Oracle join method is available to any Teleport process running on an
+OCI Compute instance.
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+- An OCI Compute instance to host a Teleport service.
+- (!docs/pages/includes/tctl.mdx!)
+
+## Step 1/5. Create the Oracle joining token
+
+Configure your Teleport Auth Service with a special dynamic token which will
+allow services from your OCI tenants to join your Teleport cluster.
+
+Under the hood, services will prove that they are running in your OCI
+tenant by sending a presigned self-authentication request to the OCI API
+for the Auth Service to execute.
+
+Create the following `token.yaml` file with an `oracle.allow` rule specifying
+the Oracle tenant(s), compartment(s), and region(s) in which your OCI
+Compute instances will run:
+
+(!docs/pages/includes/provision-token/oracle-spec.mdx!)
+
+Run the following command to create the token:
+
+```code
+$ tctl create token.yaml
+```
+
+## Step 2/5. Configure permissions
+
+### Create a dynamic group
+
+In the OCI console, navigate to
+[Identity/Domains](https://cloud.oracle.com/identity/domains).
+Select your domain, then select **Dynamic groups**. Click **Create dynamic group**.
+Create a group with the following matching rule, using the OCID of the
+compartment your instance is in:
+```code
+Any {instance.compartment.id = '<Var name="compartment-id" />'}
+```
+
+![Create dynamic group](../../../../img/oracle/oracle-join-dynamic-group@2x.png)
+
+<Notice type="tip">
+
+To minimize unnecessary permissions, configure your matching rules to match
+the rules in the token created in step 1.
+
+</Notice>
+
+### Add policy to group
```

Comment:
this explains what to do, but i think it might help to give a high-level explanation at the beginning of this step that every instance joining with this method needs this policy and why